520- Final project.R

```diff
@@ -0,0 +1,364 @@
+
+  ggplot(data, aes(x = .data[[x_var]], y = .data[[y_var]])) +
+    geom_point() +
+    geom_smooth(method = "lm", se = FALSE, color = "blue") +  # Add regression line
+    labs(title = plot_title,
+         x = x_var,
+         y = y_var,
+         log = "xy")
+  
+}
+
+
+######################### High school education and poverty ######################
+
+plot_scatter_with_lm(data = svi, 
+             x_var = "e_nohsdp", 
+             y_var = "e_pov", 
+             plot_title = "For US counties: relationship between counts of people lacking high school diploma and counts of people below 1.5 times poverty line")
+plot_scatter_with_lm(data = svi_state, 
+             x_var = "percent_nohsdp", 
+             y_var = "percent_pov", 
+             plot_title = "For US states: relationship between percent of people who lack of high school diploma and percent below 1.5 times poverty line")
+plot_scatter_with_lm(data = svi_wa, 
+             x_var = "e_nohsdp", 
+             y_var = "e_pov", 
+             plot_title = "For WA counties: relationship between counts of people lacking high school diploma and counts of people below 1.5 times poverty line")
+
+
+# Fit linear regression model for US counties
+lm_nohsdp <- lm(e_pov ~ e_nohsdp, data = svi)
+# Print the summary of the linear regression model
+summary(lm_nohsdp)
+lm1_summary <- summary(lm_nohsdp)
+print(lm1_summary)
+
+
+# Fit linear regression model for Washington
+lm_nohsdp_washington <- lm(e_pov ~ e_nohsdp, data = svi_wa)
+# Print the summary of the linear regression model for Washington
+summary(lm_nohsdp_washington)
+
+
+######################### High school education and unemployment ######################
+
+plot_scatter_with_lm(data = svi, 
+             x_var = "e_nohsdp", 
+             y_var = "e_unemp", 
+             plot_title = "For US counties: relationship between counts of people lacking high school diploma and counts of people unemployed")
+plot_scatter_with_lm(data = svi_state, 
+             x_var = "percent_nohsdp", 
+             y_var = "percent_unemp", 
+             plot_title = "For US states: relationship between percent of people who lack of high school diploma and percent unemployed")
+plot_scatter_with_lm(data = svi_wa, 
+             x_var = "e_nohsdp", 
+             y_var = "e_unemp", 
+             plot_title = "For WA counties: relationship between counts of people lacking high school diploma and counts of people unemployed")
+
+
+# Fit linear regression model
+lm_unemp <- lm(e_unemp ~ e_nohsdp, data = svi)
+# Print the summary of the linear regression model
+summary(lm_unemp)
+lm2_summary <- summary(lm_unemp)
+print(lm2_summary)
+
+
+# Fit linear regression model for WASHINGTON
+lm_nohsdp_washington <- lm(e_unemp ~ e_nohsdp, data = svi_wa)
+# Print the summary of the linear regression model for Washington
+summary(lm_nohsdp_washington)
+
+
+################# Mapping ###################
+
+# TODO: consider removing from this script

```

Comment:
I know we had talked about moving all of our functions into a separate script, and this reminder is leftover from that discussion. We can probably just delete the comment